tests/unit/pillar/test_mysql.py

```diff
@@ -13,7 +13,7 @@
 
 
 @skipIf(NO_MOCK, NO_MOCK_REASON)
-@skipIf(PY3, 'MySQL-python is not compatible with python3')
+@skipIf(mysql.HAS_MYSQL is False, 'Install MySQL bindings before running MySQL unit tests.')
```

Comment:
@isbm agree with you. I've wrote it in that way just because of this: 😄 
```
$ grep -Rn "skipIf.*HAS_" tests/ | grep "is False" | wc -l
135
$ grep -Rn "skipIf.*not.*HAS_" tests/ | wc -l
78
```
Anyway, I'll update that.
Thank you for review.